create_db.sh

```diff
@@ -1,9 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Common startup assumes bash, I guess I did this for consistency with that script. I can revert the change if you wish.